Tests/KituraTests/TestRequests.swift

```diff
@@ -69,9 +70,17 @@ class TestRequests: KituraTest {
         // Set up router for this test
         let router = Router()
 
+        struct Params: QueryParams {
+            let key1: String
+            let key2: String
+            let key3: String
+        }
+
         // Test query params
         router.get("/xyz") { request, _, next in
             let expectedQueryParams = ["key1" : "value1", "key2" : "value2", "key3" : "value3"]
+            let parsedParams = request.getQueryParameters(as: Params.self)!
```

Comment:
I think this might be better as:

```swift
guard let parsedParams = request.getQueryParameters(as: Params.self) else {
    XCTFail("Failed to get query parameters")
    return next()
}
```